app/brave_generated_resources.grd

```diff
@@ -310,6 +310,15 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
       <message name="IDS_BRAVE_SYNC_V2_MIGRATE_INFOBAR_COMMAND" desc="Command text shown to users who used a previous version of Brave Sync and need to perform setup again for the new version.">
         Begin Setup
       </message>
+      <message name="IDS_BRAVE_SYNC_ACCOUNT_DELETED_INFOBAR_MESSAGE" desc="Message text shown to users who's Sync account was deleted. In continuation goes the link IDS_BRAVE_SYNC_ACCOUNT_DELETED_INFOBAR_LINK_TEXT">
+        This Brave Sync account was deleted from other device. If you didn’t mean to remove the Sync,
```

Comment:
This string seems incomplete

Reply:
Yes, the informer consists of the message and the link which is a part of the sentence 

like  

This Brave Sync account was deleted from other device. If you didn’t mean to remove the Sync, [re-create the account](brave://settings)

https://user-images.githubusercontent.com/24739341/197198794-944e8d28-c32c-4627-9591-2d11f7dbb948.png
https://user-images.githubusercontent.com/24739341/197201801-0c87e700-0881-49bd-8108-8fc7290749b1.png

If it confuses, I will fix the decryption to have less questions from the Transifex crew.

Reply:
Yes, we should probably add a placeholder there for translators to better understand the context.

Reply:
fixed here
https://github.com/brave/brave-core/pull/13562/commits/633b9fd8629179bde34878b1f3d338a1d43c94bf

Reply:
commit changed after rebase:
01e06ea3cdf2184a4aeaa707a1e2687c951eee83